### PR TITLE
docs: add a landing page to the documentation

### PR DESCRIPTION
**Pull request by 12rambau**
A pydata-sphinx-theme based documentation needs a landing page. 

## features 

- A centered 2 column landing hero container 
- left is a bit of text + title + 3 buttons (leading nowhere at the moment) 
- right is a demo gif 

## technical addition 
- hero container required extra css rules
- demo gif is now existing in both dark and light theme
- hide the index 
- the hero container is mainly made of pure html to no break other potential builds (👋  .pdf) 

image

### CODE REVIEW

**Comment by mariobuikhuizen** [comment]
The landing page looks much better than the site-map previously! Awesome the demo gif is now dark-mode enabled!

The buttons/links don't seem to work, but I don't know if this PR is ready yet.

**Comment by 12rambau** [comment]
the button are pointing to pages that are not yet existing so I send them to "#", the PR is ready to merge !

**Comment by mariobuikhuizen** [comment]
Ok, let's wait for those pages to be available, otherwise we'll have broken links on the doc-site when we need to release from master.

**Comment by 12rambau** [comment]
require #278 to move forward

**Comment by mariobuikhuizen** [comment]
What is we removed the dead links for now, then we can merge the landing page. We'll add back the links in PR's that have the actual content?

**Comment by 12rambau** [comment]
@mariobuikhuizen I left the code of the buttons but made them disapear. In this configuration, wiring them once the other pages are ready will be super easy and we don't show to users dead links.

File: docs/index.rst [hunk 1]
```diff
@@ -1,7 +1,68 @@
...
+    <div id="hero">
+        <div id="hero-left">
+
+IpyVuetify
+----------
+
+Jupyter widgets based on Vuetify UI components
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**ipyvuetify** is a widget library for making modern looking GUI’s in Jupyter notebooks (`classic <https://github.com/jupyter/notebook>`__, `lab <https://github.com/jupyterlab/jupyterlab>`__, `lite <https://github.com/jupyterlite/jupyterlite>`__) and dashboards (`Voilà <https://github.com/voila-dashboards/voila>`__, `Voici <https://github.com/voila-dashboards/voici>`__). It’s based on the `Vuetify UI <https://v2.vuetifyjs.com/en/>`__ library. it has a lot more widgets than the standard jupyter widget library which, all of them being customizables and composables.
```
Comment (mariobuikhuizen) [inline docs/index.rst:33]:
```suggestion
**Ipyvuetify** is a widget library for making modern-looking GUIs in Jupyter notebooks (`classic <https://github.com/jupyter/notebook>`__, `lab <https://github.com/jupyterlab/jupyterlab>`__, `lite <https://github.com/jupyterlite/jupyterlite>`__) and dashboards (`Voilà <https://github.com/voila-dashboards/voila>`__, `Voici <https://github.com/voila-dashboards/voici>`__). Based on the `Vuetify UI <https://v2.vuetifyjs.com/en/>`__ library, it extends the standard Jupyter widget library with additional widgets that are more customizable and composable.
```

Comment (mariobuikhuizen) [reply docs/index.rst:33]: This fixes some spelling errors (also from parts that were in introduction before) and improved a bit by chatgpt.

**Review by mariobuikhuizen — CHANGES_REQUESTED** [review]
Looks good! Only needs some changes to the text.

File: docs/index.rst (hunk 1 above)
Comment (12rambau) [reply docs/index.rst]: perfect, I merged them in.

**Pull request merged**